addons/sale_project/wizard/project_template_create_wizard.xml

```diff
@@ -34,7 +34,12 @@
                     name="template_id"
                     string="Project Template"
                     placeholder="New project"
-                    domain="[('is_template', '=', True)]"
+                    domain="[
+                        ('is_template', '=', True),
+                        '|',
+                        ('company_id', '=', context.get('sale_company_id')),
```

Comment:
By doing that, if `sale_company_id` does not exist, we will no longer fetch project templates with company set.

Reply:
also, I guess you wanted to use `so_company_id`